msteams-platform/concepts/authentication/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Resource specific consent in Teams
+description: Describes resource specific consent to apps accessing Team's data and how to configure it.
+keywords: RSC resource specific consent
+---
+# Resource specific consent in Teams
+
+Resource-specific consent (RSC) enables team owners to consent apps accessing Teams data. Teams app can now call Teams' Graph APIs without the need of [tenant-wide admin consent](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/grant-admin-consent). These APIs include creating, renaming and deleting channels, reading channel messages, creating tabs, adding and removing members from teams. This article provides the steps to take advantage of RSC.
+
+Previously, apps calling Teams Graph APIs required a global admin to approve the app before the app could run and access team data and required very high levels of permissions that admins were reluctant to grant. Resource-specific consent eliminates this friction by allowing team owners to give consent to an app to use just their team's data, without the need for a global admin to approve the app tenant-wide. To the team owner, it will look just like any other Teams app, only with more powerful permissions. 
+
+![Consent screen.](../../assets/images/rsc/rsc-screen.png)
+
+## Prerequisites
+
+* ### Settings in Azure AD:
+By default, the RSC feature is enabled in tenants where `Users can consent to apps accessing company data on their behalf` is set to `Yes`. Team owners cannot provide consent if this setting is not enabled. 
+
+Possible values for [EnableGroupSpecificConsent](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/configure-user-consent) setting- 
```

Comment:
Both the settings in the prerequisite section are redirected to a single link now. All settings are mentioned there. (to avoid duplicates) Please refer the updated document and let me know in case of any changes.